src/utils/dispute-utils.js

```diff
@@ -208,26 +225,36 @@ export function getPhaseAndTransition(dispute, courtConfig, nowDate) {
   }
 }
 
+/**
+ * Tells the adjudication state of a round at a certain time.
+ * @param {Object} dispute Dispute querying the adjudication round of
+ * @param {Object} round The round that is being queried
+ * @param {Number} now Current time in ms
+ * @param {Object} courtConfig The court configuration
+ * @returns {Object} The adjudication phase of the requested round at the given time
+ */
 export function getAdjudicationPhase(dispute, round, now, courtConfig) {
   const {
     termDuration,
     commitTerms,
     revealTerms,
     appealTerms,
     appealConfirmationTerms,
-    maxRegularAppealRounds,
   } = courtConfig
-  const numberOfRounds = dispute.rounds.length
+
   const { draftTermId, delayedTerms, number: roundId } = round
 
   const drafTermStartTime = getTermStartTime(draftTermId, courtConfig)
+  const maxAppealReached = hasDisputeReachedMaxAppeals(dispute, courtConfig)
 
+  // Case where we are at the last possible round and voting period has not started yet
   if (
-    DisputesTypes.Phase[round.state] === DisputesTypes.Phase.Invalid &&
-    now < drafTermStartTime
+    now < drafTermStartTime &&
+    (DisputesTypes.Phase[round.state] === DisputesTypes.Phase.Invalid ||
```

Comment:
Does `DisputesTypes.Phase[round.state] === DisputesTypes.Phase.Invalid` hold for `Phase.NotStarted`? This seems like still an invalid state.

Reply:
Yes good catch. I'm refactoring this bit of code and adding more comments